content/spin/javascript-components.md

```diff
@@ -281,4 +330,4 @@ These are some of the suggested libraries that have been tested and confired to
 
 - All `spinSdk` related functions and methods can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
 - Only a subset of the browser and `Node.js` APIs are implemented.
-- The support for `Crypto` module is currently limited to only `getRandomValues`. 
\ No newline at end of file
+- The support for `Crypto` module is currently limited to only `getRandomValues`. 
```

Comment:
With `js2wasm` v0.3.0, we now support `crypto.subtle.digest` and `cryto.createHash` /`crypto.createHmac`.

Reply:
Thanks @karthik2804 
This has been added as part of latest commit.